### DISCUSSION TITLE
расширение в Google Chrome скоро "все"?

### DISCUSSION THREAD

**Discussion by drunken0master** [anticensority/runet-censorship-bypass#286] [General]
С недавних пор при входе в раздел с расширениями chrome начал сообщать о скором прекращении поддержки вашего расширения. Что гугл хочет этим сказать? Или просто стоит просто ждать другую версию расширения?
![image](https://github.com/user-attachments/assets/ac5291cc-23b7-44e2-bfcc-cd16c2155719)

**Comment by KazeKaze93** [comment]
В последней версии браузера Google Chrome [появились уведомления](https://www.playground.ru/redirect/https/www.bleepingcomputer.com/news/google/google-chrome-warns-ublock-origin-may-soon-be-disabled/) для пользователей uBlock Origin, сигнализирующие о необходимости поиска альтернативных решений для блокировки рекламы. Это связано с предстоящим прекращением поддержки расширений, работающих на базе Manifest v2.

Начиная с Chrome 127, на странице управления расширениями ("chrome://extensions/") отображается предупреждение о скором прекращении функционирования дополнений, не совместимых с третьей версией манифеста. Google планирует завершить переход на новую версию до конца текущего года. Для корпоративных пользователей Chrome с определёнными настройками предусмотрена отсрочка: они смогут использовать расширения на базе Manifest v2 до июня 2025 года.

**Reply by drunken0master** [reply]
Я уже успел прочитать о Manifest v3 и что там будет все печально с блокировщиками рекламы. Как затронет это обновление данное расширение?

**Reply by jolywho** [reply]
достаточно перейти на нормальный браузер

**Comment by KazeKaze93** [comment]
То, что третий манифест поддерживают не все расширения и это касается не
только блокировщиков?)

Пн, 5 авг. 2024 г. в 21:05, Art ***@***.***>:

**Reply by drunken0master** [reply]
именно, и волнует конкретно поддержка этого расширения.  Благодаря ему можно не задумываться ни о каком впн.

**Comment by owl5053** [comment]
чтобы google chrome не пытался остановить/удалить эти расширения, сделайте .reg файл с содержимым:

> Windows Registry Editor Version 5.00
> 
> [HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Google\Chrome]
> "ExtensionManifestV2Availability"=dword:00000002


примените и перезапустите Google Chrome
